dev-python/rdflib/rdflib-4.2.2.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="Python library for working with RDF"
+DESCRIPTION="RDFLib is a Python library for working with RDF, a simple \
+yet powerful language for representing information as graphs."
+HOMEPAGE="https://github.com/RDFLib/rdflib"
+COPYRIGHT="2002-2009 Daniel Krech"
+LICENSE="Public Domain"
```

Comment:
I don't think [this](https://github.com/RDFLib/rdflib/blob/1503dae6049f0d9b14d9d7f884d8de4cb38b39a3/LICENSE) is "Public Domain"